src/wp-includes/class-wp-scripts.php

```diff
@@ -882,6 +883,18 @@ private function get_intended_strategy( $handle ) {
 		return isset( $script_args['strategy'] ) ? $script_args['strategy'] : false;
 	}
 
+	/**
+	 * Check if the strategy passed is one of the delayed strategies.
```

Comment:
```suggestion
	 * Check if the strategy passed is a non-blocking strategy.
```

Reply:
@joemcgill @kt-12 I've addressed this in a recent commit, I am going to merge this PR.